src/pybind/mgr/mgr_module.py

```diff
@@ -427,7 +428,7 @@ def get_module_option(self, key, default=None):
         :param default: the default value of the config if it is not found
         :return: str
         """
-        r = self._ceph_get_module_option(key)
+        r = self._ceph_get_module_option(key, "")
         if r is None:
             final_key = key.split('/')[-1]
```

Comment:
As we are catching localized settings in a new chunk of code. Shouldn't we remove this?